recipe/conda_build_config.yaml

```
@@ -26,8 +26,8 @@ vsver:
 runtime_version:
  # the azure/github images only contain the latest minor version per vs-line;
  # if there is a use-case for having older minor versions, they can be added when requested.
- - 14.40.33810
- - 14.40.33810
+ - 14.41.34120
+ - 14.41.34120
```

my gut-feel says that we need all of them, especially looking at this:
```
************************************
MSVC Runtimes version: 14.40.33810.0
************************************

Traceback (most recent call last):
  File "C:\Miniforge\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Miniforge\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "D:\bld\vs2022_1725452832861\work\vc_repack.py", line 377, in <module>
    main()
  File "D:\bld\vs2022_1725452832861\work\vc_repack.py", line 355, in main
    unpack_exe(exe_path, env, args.version)
  File "D:\bld\vs2022_1725452832861\work\vc_repack.py", line 278, in unpack_exe
    raise RuntimeError(
RuntimeError: Wanted version 14.41.34120, found 14.40.33810
```
i haven't dug into why the duplications are necessary but it's been the same in the past
```suggestion
 - 14.41.34120
 - 14.41.34120
 - 14.41.34120
 - 14.41.34120
 - 14.40.33810
 - 14.40.33810
```

---

@conda-forge-admin, please rerender

---

This would only work if I add changes to the other places as well within the recipe, since all must have the same amount of items, and unfortunately at this point I would just be randomly guessing.